airflow/providers/amazon/aws/operators/emr.py

```diff
@@ -1035,14 +1036,15 @@ def execute(self, context: Context) -> dict:
         if response["ResponseMetadata"]["HTTPStatusCode"] != 200:
             raise AirflowException(f"EMR serverless job failed to start: {response}")
 
-        self.log.info("EMR serverless job started: %s", response["jobRunId"])
+        self.job_id = response["jobRunId"]
+        self.log.info("EMR serverless job started: %s", self.job_id)
         if self.wait_for_completion:
             # This should be replaced with a boto waiter when available.
```

Comment:
WDYT about adding a log statement here while waiting for the job to get cancelled. It would give an impression to the users that it is stuck if there is no log.

Reply:
Actually, just double-checked and noticed in the `on_kill` section I do have a log statement that prints "Stopping job run".